src/main/kotlin/com/ajax/cryptocurrency/model/Cryptocurrency.kt

```diff
@@ -8,7 +8,7 @@ import java.time.LocalDateTime
 @Document(collection = "statistic")
 data class Cryptocurrency(
     @Id
-    var id: ObjectId = ObjectId(),
+    var id: String?,
```

Comment:
@OlehSarapuk what would you say?

Reply:
moved back to ObjectId type

Reply:
in previous situation it is better to use lateinit instead of nullable type